torch/_torch_docs.py

```diff
@@ -5027,6 +5027,63 @@ def merge_dicts(*dicts):
     tensor([[False, False], [True, False]])
 """.format(**common_args))
 
+add_docstr(torch.lu_unpack, r"""
+lu_unpack(LU_data, LU_pivots, unpack_data=True, unpack_pivots=True) -> (Tensor, Tensor, Tensor)
```

Comment:
This function supports the out variant now. It should be added to the documentation.